File: js/get_message_status.js
const url =
  '<REPLACE_WITH_PROD_URL>/v1/messages/<YOUR MESSAGE ID>/status?token=<YOUR TOKEN>'

const fetchData = async () => {
  try {
    const response = await fetch(url, {
      method: 'GET',
    })

    const data = await response.json()
    console.log(JSON.stringify(data))
  } catch (error) {
    console.log(error)
  }
}

fetchData()

File: vb.net/get_message_status.vb
Dim client As New RestClient("<REPLACE_WITH_PROD_URL>/v1/message/<REPLACE_WITH_YOUR_MESSAGE_ID>/status?token=<YOUR TOKEN>")
client.Timeout = -1

Dim request As New RestRequest(Method.GET)

Dim body As String = ""
request.AddParameter("text/plain", body, ParameterType.RequestBody)

Dim response As IRestResponse = client.Execute(request)

Console.WriteLine(response.Content)
